docs/2.9.0/docs/query/pqs-user-guide.rst

```diff
@@ -1,466 +1,1741 @@
+       Suppressed: io.grpc.StatusException: UNAVAILABLE: io exception
+           Suppressed: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: localhost/[0:0:0:0:0:0:0:1]:6865
+               Suppressed: java.net.ConnectException: Connection refused application=scribe
+   12:52:51.237 I [zio-fiber-0] com.digitalasset.scribe.pipeline.Retry.retryRecoverable:48 Recoverable GRPC exception. Attempt 3, unstable for 34 seconds. Remaining attempts: 40. Remaining time: 9 minutes 25 seconds. Exception in thread "zio-fiber-" java.lang.Throwable: Recoverable GRPC exception.
+       Suppressed: io.grpc.StatusException: UNAVAILABLE: io exception
+           Suppressed: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: localhost/[0:0:0:0:0:0:0:1]:6865
+               Suppressed: java.net.ConnectException: Connection refused application=scribe
+   12:53:33.473 I [zio-fiber-0] com.digitalasset.scribe.pipeline.Retry.retryRecoverable:48 Recoverable GRPC exception. Attempt 4, unstable for 1 minute 16 seconds. Remaining attempts: 39. Remaining time: 8 minutes 43 seconds. Exception in thread "zio-fiber-" java.lang.Throwable: Recoverable GRPC exception.
+       Suppressed: io.grpc.StatusException: UNAVAILABLE: io exception
+           Suppressed: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: localhost/[0:0:0:0:0:0:0:1]:6865
+               Suppressed: java.net.ConnectException: Connection refused application=scribe
+   12:54:36.328 I [zio-fiber-0] com.digitalasset.scribe.pipeline.Retry.retryRecoverable:48 Recoverable JDBC exception. Attempt 5, unstable for 2 minutes 19 seconds. Remaining attempts: 38. Remaining time: 7 minutes 40 seconds. Exception in thread "zio-fiber-" java.lang.Throwable: Recoverable JDBC exception.
+       Suppressed: org.postgresql.util.PSQLException: Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.
+           Suppressed: java.net.ConnectException: Connection refused application=scribe
+
+Metrics
+~~~~~~~
+
+The following metrics are available to monitor stability of PQS's
+dependencies. See the `observability <#observability>`__ section for
+more details on general observability:
+
+.. code:: text
+
+   # TYPE app_restarts_total counter
+   # HELP app_restarts_total Number of total app restarts due to recoverable errors
+   app_restarts_total{,exception="Recoverable GRPC exception."} 5.0
+
+   # TYPE grpc_up gauge
+   # HELP grpc_up Grpc channel is up
+   grpc_up{} 1.0
+
+   # TYPE jdbc_conn_pool_up gauge
+   # HELP jdbc_conn_pool_up JDBC connection pool is up
+   jdbc_conn_pool_up{} 1.0
+
+Instability counters reset
```

Comment:
Are these instability counters helpful for us to debug issues?  Isn't this too much to ask users to understand?  Is this best to put in an "Advanced monitoring" or "Debugging" section?

Reply:
Of all the metrics this is a prominant one as network instability is something everyone is going to want to monitor. I wouldn't categorize them as any more advanced than any other metric?